vimeo-networking/src/main/java/com/vimeo/networking/VimeoService.java

```diff
@@ -392,6 +395,23 @@
     // </editor-fold>
 
     // -----------------------------------------------------------------------------------------------------
+    // IAP
+    // -----------------------------------------------------------------------------------------------------
+    // <editor-fold desc="In-App Purchases">
+    @GET("products")
+    Call<Products> getProducts(@Header("Authorization") String authHeader);
+
+    @GET("products/{product_id}")
+    Call<Product> getProduct(@Header("Authorization") String authHeader,
+                             @Path("product_id") String productId);
+
+    @PUT("/me/purchases/{product_id}")
```

Comment:
This endpoint probably should not be hard-coded here. Is it available in the metadata of the User object?

Reply:
Good question.  There was no mention of this in any of the docs I've reviewed yet, but that would probably make sense.  I've asked the BE dev.